doc/nrf/index.rst

```diff
@@ -36,3 +36,5 @@ Documentation for different versions of the |NCS| is available at the following
    scripts
 
 ..   cheat_sheet
+
+..   sample_boardnames.rst
```

Comment:
This seems to work, so it's okay for now, but ideally we should change the file name to sample_boardnames.txt (like our other include files). Then it won't be copied and included as rst file.